exasol_advanced_analytics_framework/temporary_operations/temporary_bucketfs_file_manager.py

```diff
@@ -0,0 +1,19 @@
+
+
+class TemporaryBucketFSFileManager:
+    def __init__(self):
+        pass
+
+    def store_states(self):
```

Comment:
what do you understand under states, do you mean the user can directly store objects maybe with a key, this might be a useful functionality, but I would call it store/load object

Reply:
agree, naming as load/store objects makes more sense